### PR TITLE
[CoreGraphics] Add support for xcode 15 beta 6.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/CoreGraphics/CGContext.cs [hunk 1]
```diff
@@ -1420,6 +1420,21 @@ public CGBitmapContext AsBitmapContext ()
...
+		[SupportedOSPlatform ("maccatalyst17.0")]
+		[SupportedOSPlatform ("macos14.0")]
+		[SupportedOSPlatform ("tvos17.0")]
+#else
+		[Mac (14, 0), iOS (17, 0), TV (17, 0), MacCatalyst (17, 0), Watch (10, 0)]
+#endif
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern void CGContextDrawConicGradient (/* CGContext */ IntPtr context, /*[NullAllowed] CGGradient*/ IntPtr gradient, CGPoint center, nfloat angle);
+
+		public void CGContextDrawConicGradient (CGGradient gradient, CGPoint point, nfloat angle) =>
```
Comment (dalexsoto) [inline src/CoreGraphics/CGContext.cs:1435]:
```suggestion
		public void DrawConicGradient (CGGradient gradient, CGPoint point, nfloat angle) =>
```

**Review by dalexsoto — APPROVED** [review]
after name fix

File: src/CoreGraphics/CGContext.cs [hunk 2]
```diff
@@ -1420,6 +1420,21 @@ public CGBitmapContext AsBitmapContext ()
...
+			CGContextDrawConicGradient (Handle, gradient?.Handle ?? IntPtr.Zero, point, angle);
```
Comment (rolfbjarne) [inline src/CoreGraphics/CGContext.cs:1436]:
```suggestion
			CGContextDrawConicGradient (Handle, gradient.GetHandle (), point, angle);
```

File: src/CoreGraphics/CGContext.cs (hunk 1 above)
Comment (rolfbjarne) [inline src/CoreGraphics/CGContext.cs:1435]:
```suggestion
		public void CGContextDrawConicGradient (CGGradient? gradient, CGPoint point, nfloat angle) =>
```

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by tj-devel709 — APPROVED** [review]
After other comments

File: src/CoreGraphics/CGContext.cs [hunk 3]
```diff
@@ -1420,6 +1420,21 @@ public CGBitmapContext AsBitmapContext ()
...
+		[SupportedOSPlatform ("maccatalyst17.0")]
+		[SupportedOSPlatform ("macos14.0")]
+		[SupportedOSPlatform ("tvos17.0")]
+#else
+		[Mac (14, 0), iOS (17, 0), TV (17, 0), MacCatalyst (17, 0), Watch (10, 0)]
+#endif
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern void CGContextDrawConicGradient (/* CGContext */ IntPtr context, /*[NullAllowed] CGGradient*/ IntPtr gradient, CGPoint center, nfloat angle);
+
+		public void DrawConicGradient (CGGradient gradient, CGPoint point, nfloat angle) =>
```
Comment (mandel-macaque) [inline src/CoreGraphics/CGContext.cs:1435]:
```suggestion
		public void DrawConicGradient (CGGradient? gradient, CGPoint point, nfloat angle) =>
```

File: src/CoreGraphics/CGContext.cs [hunk 4]
```diff
@@ -1420,6 +1420,21 @@ public CGBitmapContext AsBitmapContext ()
 		{
 			return new CGBitmapContext (Handle, false);
 		}
+
+#if NET
+		[SupportedOSPlatform ("ios17.0")]
+		[SupportedOSPlatform ("maccatalyst17.0")]
+		[SupportedOSPlatform ("macos14.0")]
+		[SupportedOSPlatform ("tvos17.0")]
```
Comment (haritha-mohan) [inline src/CoreGraphics/CGContext.cs:1428]: why the diff in how we define the platform attrs for .net vs legacy? iirc these bindings that are within a framework directory go through one less pass through the generator, so assuming that has something to do with it?

**Review by rolfbjarne — APPROVED** [review]

File: src/CoreGraphics/CGContext.cs (hunk 4 above)
Comment (mandel-macaque) [reply src/CoreGraphics/CGContext.cs:1428]: This is manually generated code, both need to be added. in the generator written code this is added. Look at the .g.cs files.

**Pull request merged**